### PR TITLE
Add mobile:swipe command

### PR DESCRIPTION
**Pull request by dpgraham**
Espresso has swipeUp, swipeDown, swipeLeft and swipeRight commands and they're mentioned in the cheatsheet so they must be important. Added them to the server.

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ commands.mobilePerformEditorAction = async function (opts = {}) {
   return await this.espresso.jwproxy.command('/appium/device/perform_editor_action', 'POST', {action});
 };
 
+commands.mobileSwipe = async function (opts = {}) {
+  const {direction, element} = opts;
```
Comment (mykola-mokhnach) [inline lib/commands/general.js]: should we unwrap the element first like we do in other drivers?

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/MobileSwipe.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+import static android.support.test.espresso.action.ViewActions.swipeDown;
+import static android.support.test.espresso.action.ViewActions.swipeLeft;
+import static android.support.test.espresso.action.ViewActions.swipeRight;
+import static android.support.test.espresso.action.ViewActions.swipeUp;
+
+public class MobileSwipe implements RequestHandler<MobileSwipeParams, Void> {
+
+    @Override
+    @Nullable
```
Comment (mykola-mokhnach) [inline espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/MobileSwipe.java]: it makes not much sense to mark Void method as nullable ;)

File: lib/commands/general.js (hunk 1 above)
Comment (dpgraham) [reply lib/commands/general.js]: Is there an example of this. The only one I could find was in `appium-uiautomator2-driver`

```javascript
commands.mobileScrollBackTo = async function (opts) {
  const {elementId, elementToId} = opts;
  return await this.uiautomator2.jwproxy.command(`/appium/element/${elementId}/scroll_to/${elementToId}`, 'POST', {});
};
```

Comment (mykola-mokhnach) [reply lib/commands/general.js]: Very often I see that https://github.com/appium/appium-support/blob/db17f4f827c4e5e53b03481ae428cfcae25e9ff5/lib/util.js#L113 is used for such purpose. @imurchie Can you please provide more details on where this helper should be used?

**Comment by saikrishna321** [comment]
https://github.com/appium/appium-espresso-driver/issues/259

**Pull request merged**